### PR TITLE
Even more quick tests

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: test_data/std/binary_test.asm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 use std::machines::binary::Binary;
 
 
-machine Main with degree: 262144 {
+machine Main with degree: 196608 {
```
Comment (leonardoalt) [inline test_data/std/binary_test.asm:4]: I'm not sure this makes a difference?

**Review by leonardoalt — DISMISSED** [review]

File: test_data/std/binary_test.asm (hunk 1 above)
Comment (chriseth) [reply test_data/std/binary_test.asm:4]: ah wait, does this even work if it's not a power of two?

Comment (chriseth) [reply test_data/std/binary_test.asm:4]: looks like it does, even in the backends?

Comment (leonardoalt) [reply test_data/std/binary_test.asm:4]: It gets padded to the next power of 2

**Review by leonardoalt — APPROVED** [review]

File: test_data/std/binary_test.asm (hunk 1 above)
Comment (chriseth) [reply test_data/std/binary_test.asm:4]: Then it's a nice test...

**Pull request merged**